app/views/content/train-to-be-a-teacher/accessibility-adjustments.md

```diff
@@ -0,0 +1,61 @@
+---
+title: "Adjustments to help you train to teach"
+heading: "Adjustments to help you train to teach"
+subcategory: Postgraduate teacher training
+description: |-
+  Find out about adjustments teacher training providers can make to help you train.
+cta_adviser:
+  adviser:
+    text: Talk to an adviser with years of teaching experience about your different training and funding options. Chat by phone, text or email, as little or as often as you need.
+calls_to_action:
+  find:
+    name: find
+navigation: 20.25
+navigation_title: Adjustments to help you train
+navigation_description: Find out about adjustments teacher training providers can make to help you train.
+cta_arrow_link:
+  financial-support:
+    link_target: "/funding-and-support/if-youre-disabled"
+    link_text: "Find out more about financial support if you're disabled"
+---
+
+Teacher training providers can make adjustments if you need supoprt to become a teacher because you're neurodiverse, have a long-term physical or mental health condition, or have any other accessibility needs. 
+
+The experience and perspective of a diverse workforce is invaluable in schools. These adjustments help make sure that anyone interested in becoming a teacher feels confident and comfortable to be the best teacher they can be.
+
+## Adjustments when you apply 
+
+During the application process, providers could make adjustments such as: 
+
+* provide an accessible interview location  
+* use an alternative communication method 
+* give extra details about the interview format and venue
+
+You can tell providers about what you need when you apply. 
+
+## Adjustments during your teacher training 
+
+During your teacher training, providers could make adjustments such as: 
+
+* adjust your timetable to suit your needs 
+* provide copies of presentation materials before training sessions 
+* print or photocopy materials on coloured paper 
+* put you in touch with support staff if you have a mental health condition 
+* make sure training and teaching areas are wheelchair accessible 
+
+## Speaking to providers about adjustments 
+
+If you think you’ll need adjustments, you can either contact your training provider directly, or you can also ask for support when you apply for teacher training. 
+
+If you’re not sure what adjustments you may need, speaking to providers can be an opportunity to get more details on the course structure and expectations. This can help you to identify any situations where you may need adjustments. Providers may also have had experience supporting other trainees and have their own suggestions of what you may find helpful.  
+
+You can also explore [Disability Rights UK’s students guide for telling people you’re disabled](https://www.disabilityrightsuk.org/resources/telling-people-you%E2%80%99re-disabled-clear-and-easy-guide-students).  
```

Comment:
```suggestion
You can also explore [Disability Rights UK’s guide to telling people you’re a disabled student](https://www.disabilityrightsuk.org/resources/telling-people-you%E2%80%99re-disabled-clear-and-easy-guide-students).  
```
I've reworded this but feel free to change - however, `students` is not right as it is missing an apostrophe!